test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java

```diff
@@ -251,19 +251,18 @@ public static void killAppLauncherProcess(JPackageCommand cmd,
     }
 
     private static long[] findAppLauncherPIDs(JPackageCommand cmd, String launcherName) {
-        // Get the list of PIDs and PPIDs of app launcher processes.
+        // Get the list of PIDs and PPIDs of app launcher processes. Run setWinEnableUTF8(true) for JDK-8344275.
         // wmic process where (name = "foo.exe") get ProcessID,ParentProcessID
         List<String> output = Executor.of("wmic", "process", "where", "(name",
                 "=",
                 "\"" + cmd.appLauncherPath(launcherName).getFileName().toString() + "\"",
-                ")", "get", "ProcessID,ParentProcessID").dumpOutput(true).
-                saveOutput().executeAndGetOutput();
-
-        if ("No Instance(s) Available.".equals(output.getFirst().trim())) {
-            return new long[0];
+                ")", "get", "ProcessID,ParentProcessID").dumpOutput(true).saveOutput().
+                setWinEnableUTF8(true).executeAndGetOutput();
+        if ("No Instance(s) Available.".equals(output.get(1).trim())) {
+                return new long[0];
         }
 
-        String[] headers = Stream.of(output.getFirst().split("\\s+", 2)).map(
+        String[] headers = Stream.of(output.get(1).split("\\s+", 2)).map(
```

Comment:
Why look up the headers in the _second_ line of the output?

Reply:
> I'd run suppress the output of "chcp" command: chcp 65001 >nul 2>&1. This will make its usage transparent for the clients of the "Executor" class.

That's true. I added `> nul2 > & 1` to suppress output of "chcp" command.